js/player.js

```diff
@@ -23,6 +23,61 @@ function createPlayerPicture() {
   return embed;
 }
 
+function runTp(){
+	console.log("tp");
+	var isPathValid = true;
+	var isTpValid = true;
+	var TpPath = document.querySelector(".pathIn").value.split(",");
+	if(TpPath === []){
+		swal("Please provide a path.");
```

Comment:
we can use the swal error prompt.
you can pass 3 param in the same function. one with title. one with type. one with this message. 

refer their documentation.